### ISSUE TITLE
Pattern for move/resize

### ISSUE THREAD

**Issue by jnurthen** [w3c/aria-practices#681]
from https://www.w3.org/Bugs/Public/show_bug.cgi?id=29326

> For those patterns which need Move/resize need a new pattern which can be referenced
> 
> The following patterns may need this
> * Dialog
> * Dialog (non-modal)
> * Column Headers
> * Date Picker
> * Media Player?

**Comment by cscheffauer** [comment]
Agreed, the HTML native textarea element seems not to be accessible when resizing.
Interested in the outcome of this.
